src/server.js

```diff
@@ -7,14 +7,10 @@ const PORT = 3000;
 
 app.use(express.static('./'));
 
-
 app.get('/', (req, res) => {
-  // res.send('Content-Type', 'aplication/json');
-  res.sendFile(path.join(__dirname, '/', 'index.html'))
+    res.sendFile(path.join(__dirname, '/', 'index.pug'))
```

Comment:
Отдать нужно файл index.html из папки dist
```suggestion
    res.sendFile(path.join(__dirname, '/dist', 'index.html'))
```

Reply:
исправил